apps/nestjs-wallet/src/vc-api/credentials/dtos/verification-result.dto.ts

```diff
@@ -1,10 +1,11 @@
 import { IsString, IsArray } from 'class-validator';
+import { VerificationResult } from '../types/verification-result';
 
 /**
  * A response object from verification of a credential or a presentation.
  * https://w3c-ccg.github.io/vc-api/verifier.html
  */
-export class VerifyProofResponseDto {
+export class VerificationResultDto implements VerificationResult {
```

Comment:
I think that properties from this class are missing `ApiProperty` decorator from `swagger` package.

Reply:
The app has the CLI plugin turned on so I don't think this is necessary? https://docs.nestjs.com/openapi/cli-plugin#overview

Reply:
fair enough